### DISCUSSION TITLE
Not able to create organization and make payment

### DISCUSSION THREAD

**Discussion by lighthouse-learning** [community/community#46476] [New to GitHub]
Hi

I am trying to add a new organization to my account but I am getting a generic error mentioning that payment cannot be processed.
Any help in this regard would be appreciated.

Regards
Piyush

![image](https://user-images.githubusercontent.com/124236455/216899187-e299dddf-1a2e-4c3e-a13a-4621d2aea5e4.png)

**Comment by jge162** [comment]
Hello @lighthouse-learning you need to reach out to [GitHub Support](https://support.github.com/contact/account?tags=hubberfy_account) as we can only discuss potential remedies but seems like you need help we cannot assist with here. Good luck.

**Comment by shayan1234554321** [comment]
It is likely that there is an issue with the payment method you are trying to use to add the new organization to your GitHub account 🙀.

Here are some steps you can try to resolve the issue: 👉🏻

1. Check if your payment method is up to date and has sufficient funds.
2. Try adding a different payment method.
3. If you're still having trouble, contact GitHub support for further assistance.

Hope that helps 👆🏻
